minishift/openshift/auth.app.yaml

```diff
@@ -75,6 +75,11 @@ objects:
               configMapKeyRef:
                 name: ${SERVICE_NAME}
                 key: notification.serviceurl
+          - name: AUTH_UI_URL
```

Comment:
if we really need such a URL configured in `auth` (which I'm not really a fan of), then we'll also have to declare it in https://github.com/fabric8-services/fabric8-auth/blob/master/openshift/auth.app.yaml#L31